### PR TITLE
Convert ThankYou screens from class based to functional components

### PR DESCRIPTION
**Pull request by juliatan**
# Description

- Updated the UK, US and SE ThankYou screens from class based to functional components as part of our plans to modernise the code base.
- Took the opportunity to remove redundant code whilst I was updating these files.
- **Question: I removed the concept of `@lazyInject(Services.Consent)` and  `private consentService: IConsentService;` which was present in the UK version. What are these for, and do we need it?**

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

n/a

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [x] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

n/a

### CODE REVIEW

File: src/features/thank-you/ThankYouUKScreen.tsx [hunk 1]
```diff
@@ -3,111 +3,96 @@ import { Header } from '@covid/components/Screen';
...
-};
-
-const initialState = {
-  askForRating: false,
-  shouldShowReminders: false,
-  showTimelineCard: false,
-};
-
-export default class ThankYouUKScreen extends Component<RenderProps, State> {
-  @lazyInject(Services.Consent)
```
Comment (juliatan) [inline src/features/thank-you/ThankYouUKScreen.tsx:42]: Is this line and the following line safe to delete?

Comment (aikewoody) [reply src/features/thank-you/ThankYouUKScreen.tsx:42]: Yes it is because the variable `consentService` was not used in the file anymore

**Review by bionara — APPROVED** [review]
Looks good! @selvach do we need a ThankYouESScreen (for Spanish)? Maybe we can push to nuke the Spanish translations?

**Comment by aikewoody** [comment]
The newly made FCs look good 👌 
But tbh I haven't compared them thoroughly enough with the class based components to check for regression.

**Pull request merged**